core/templates/pages/exploration-editor-page/services/change-list.service.ts

```diff
@@ -313,12 +313,32 @@ export class ChangeListService {
 
   /**
    * Saves a change dict that represents marking a translation as needing
-   * update.
+   * update in a particular language.
    *
    * @param {string} contentId - The content id of the translated content.
    * @param {string} languageCode - The language code.
    * @param {string} stateName - The current state name.
    */
+  markTranslationAsNeedingUpdate(
```

Comment:
I think we should write unit tests for this function

Reply:
Yep, done!